vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -0,0 +1,141 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Hub
+## Overview
+
+The Entando Hub enables a team to share components across Entando Applications. It can be installed in Entando 6.3.2 or higher and includes API-level integration with the Entando 7.0 App Builder.
+
+The Hub allows users to:
+
+- Centralize components and business capabilities for use across teams, groups, or clients.
+- Publish, manage and communicate component features, versions and metadata.
+- Perform business-level assessment of component readiness. 
+
+This tutorial covers:
+
+1. [Installation](#installation)
+2. [Configuration](#configuration)
+3. [Using the Hub](#using-the-hub)
+4. [Application Details](#application-details)
+5. [Resources](#resources)
+## Installation
+
+The Hub is installed using the Entando Component Repository (ECR) and two Entando Bundles.  The first bundle contains the Hub micro frontends and microservices, and the second sets up the initial content and pages for the Hub user experience.
+
+### Prerequisites
+
+- An Entando Application on any Kubernetes provider. Follow the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
+- The ent command line tool, installed and connected to your Kubernetes instance.
+
+### Installation Steps
+
+1. Apply the Custom Resource Definitions for the Hub component bundles. You'll need to adjust the `-n entando` option in each command to match your namespace or project.
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-hub-application-bundle.git -d | ent kubectl apply -n entando -f -
+```
+```
+ent bundler from-git -r https://github.com/entando-samples/entando-hub-content-bundle.git -d | ent kubectl apply -n entando -f -
+```
+
+2. Log into your App Builder instance.
+
+3. Select `Repository` from the menu on the left. Your bundles will be visible in the repository as shown in the screenshot below.
+
+![hub-install.png](./hub-images/hub-install.png)
+
+4. Select `Install` for each bundle, where order of installation is important. The `entando-hub-bundle` will need to be installed first, as it provides the `entando-hub-content-bundle` with MFEs to set up each of the pages. An installation can take several minutes while the application downloads the Linux images for the microservices and installs the related assets.
+
+:::warning
+(Entando 6.3.2) There is a cache issue when deploying the Hub bundles where not all widgets or MFEs initially appear on some pages, particularly the Dashboard page. Restarting the quickstart-server pod (which holds the Entando App Engine) will clear the cache. This is only necessary on the initial install.
+:::
+
+5. The Hub can be accessed from the App Builder by navigating to `Pages → Management`, finding `Hub Test` **change** in the page tree, and clicking `View Published Page` from its actions.
```

Comment:
Please create a ticket for the 2nd round on this. That will include at least the actual page name and final screenshots with Search included. You can use it to collect whatever we know we're lacking.